index.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html>
+    <head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio project</title>    
+  <link rel="stylesheet" href="styles.css">
+  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+    <body>
+        <header>
+            <nav>
+                <ul>
+              <li><a class="logolink" href="">My Logo</a></li>
+              <li class="leftli"><a href=""><img src="hammenu.png" alt="ham" width="20px" height="18px"></a></li>
+                </ul>
+            </nav>
+        </header>
+        <div class="headline">
+            <div class="textdiv"><h1>I’m Alexandra <br>
+                Glad to see you!</h1>
+                <p>I’m a software developer! I can help you build a product , feature or website Look through 
+                  some of my work and experience! If you like what you see and have a project you need coded,
+                   don’t hestiate to contact me.</p>
+                  <p style="color: #7F8CFF;">LET’S CONNECT</p>
```

Comment:
- [ ] The title of this text must use a header tag ie. (`h1,h2,h3 ..)` and not a span tag. This is because it is a subtitle of the section, not a span text :100:

Reply:
changes made